inc/template-tags.php

```diff
@@ -114,7 +114,7 @@ function twentyseventeen_edit_link( $id ) {
 	} elseif ( get_post( $id ) ) {
 		$type = __( 'Post', 'twentyseventeen' );
 	}
-	$link = edit_post_link(
```

Comment:
@josephfusco Thank you so much for your tip, I tried to do that, but the both commits appear in pull request so I closed the pull request (  https://github.com/WordPress/twentyseventeen/pull/104 ), I am not so good at git :( .
Anyway I believe it is mistake in the function twentyseventeen_edit_link ( https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L116 ), I think it is just duplicate function of twentyseventeen_edit_post_link ( https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L58 ).